docs/development/pigs/pig-xxx.rst

```diff
@@ -0,0 +1,182 @@
+`DataStoreObservation.observation_live_time_duration` were updated accordingly.
+These new `~gammapy.data.ObservationList` can then be passed on to the
+*standard* spectrum analysis with the `gammapy.background` and
+`gammapy.spectrum` modules.
+
+This analysis pipeline should be done outside of the LightCurve class, for
+example by `gammapy.scripts.SpectrumAnalysisIACT`.
+The arguments for this pipeline should be provided in form of a *config dict*
+or possibly a txt file.
+One could think about providing a list of *config dict* or txt files to
+have more flexibility when performing the pipeline in each of the time bins.
+
+Instead of performing a spectral fit in each time bin, the user should also
+have the option to provide a fixed spectral shape, omit the fit and compute the
+integral flux the way it is done now, by scaling the expected counts with
+respect to the excess events, which should be much faster.
+
+
+Preparing the ObservationList for each time bin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The general idea is to write a new `~gammapy.data.ObservationList` for each
+time bin to disk, so that they can be easily read-in by the DataStore class
+with, for example, its `.from_dir()` method.
+This would avoid memory issues for large data sets and the user also gets the
+possibility to easily repeat a spectral analysis in one of the light curve
+bins.
+For this procedure `~gammapy.data.ObservationList` should get a method like 
+`.write_to_disk(outdir)`, or the `~gammapy.data.DataStore` object should get a
+method `.write_list_to_disk(obslist, outdir)` (invocation vs inspection).
+
+Depending on the modifications planned for the `gammapy.data.data_store`
+module, one could also think about providing the user with the option to do
+everything in-memory, which could especially be useful for playing 
+around/testing.
+
+The preparation of the new `~gammapy.data.ObservationList` for each time bin
+would then basically consist of 3 steps:
+ * The `LightCurveEstimator` class figures out which observation belongs to
+   which light curve bin and creates a new `~gammapy.data.ObservationList` with
+   these observations for each time bin;
+ * The new `~gammapy.data.ObservationList` are written to disk in a new folder
+   structure specified by the user;
+ * The LightCurve class modifies the event lists of the first and last (in 
+   time) observation in the respective `~gammapy.data.ObservationList` and
+   updates their `observation_live_time_duration` (what about the `GTI` s?).
+
+
+Calling the spectral pipeline for each time bin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After preparing the observations and writing them to disk, a loop over the
+light curve bins should create a `~gammapy.data.DataStore` object -> 
+`~gammapy.data.ObservationList` and pass it on to the spectral pipeline (e.g.
+`gammapy.scripts.SpectrumAnalysisIACT`).
+Maybe the option to omit the spectral fit and scale a fixed spectrum to the
```

Comment:
Again I think the logic is flawed here.

The first step is to produce reduced datasets and IRFs. So you simply want to perform spectral extraction. 

Spectral fitting should be done later so that any time rebidding can be performed on the fly.

This approach would force the user to rerun each time he want to change the bin size!